proxy/src/auth/backend.rs

```diff
@@ -161,6 +194,7 @@ impl ComputeUserInfo {
 pub enum ComputeCredentialKeys {
     Password(Vec<u8>),
     AuthKeys(AuthKeys),
+    None,
```

Comment:
Seems like CCKs are already optional. Why is it better to explicitly specify `Some(CCK::None)`?